genesyscloud/process_automation_trigger/resource_genesyscloud_processautomation_trigger.go

```diff
@@ -329,10 +349,22 @@ func buildTarget(d *schema.ResourceData) *Target {
 			targetType := targetMap["type"].(string)
 			id := targetMap["id"].(string)
 
-			return &Target{
+			target := &Target{
 				Type: &targetType,
 				Id:   &id,
 			}
+
+			workflowTargetSettingsInput := targetMap["workflow_target_settings"].(*schema.Set).List()
+			workflowTargetSettingsInputMap := workflowTargetSettingsInput[0].(map[string]interface{})
```

Comment:
It might be moving this inside the if statement. If the len of the slice == 0, casting the first item to a map could cause a the program to panic

Reply:
good call, fixed